.github/workflows/announcements.yml

```diff
@@ -0,0 +1,35 @@
+name: Fetch announcements from issue
+on: issue_comment
+jobs:
+  issue_commented:
+    # This job only runs for issue comments
+    name: Update Announcements
+    if: ${{ (!github.event.issue.pull_request) && github.event.issue.number == 181 }}
```

Comment:
Because the variables inside the `${{ }}` are substituted with their values and then executed they are subject to an injection attack. See the [Security Hardening guides](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections).

Although I don't think in our case anyone will be able to inject anything into `github.event.issue.number` it might just be good practice to move it into an [environment variable](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections).

Reply:
Do you mean move `181` as an environment variable, or do we pass in the `github.event.issue.number` as an environment variable into the python script

Reply:
Is it also possible to also limit this to trigger if someone we trust has posted an issue? Otherwise random users can comment on the issue and trigger this workflow, which will eventually trigger a build and eat up our build quota.

I found an [example payload](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-example-when-someone-edits-an-issue) of the `github.event` payload when someone updates an issue. Looks like we'll get some metadata about the user who made the change. Maybe the `site_admin` property will help?

Reply:
> Do you mean move `181` as an environment variable, or do we pass in the `github.event.issue.number` as an environment variable into the python script

`github.event.issue.number`. These are the variables that will be substituted and can thus be substituted by some bash script. In this case I don't think anyone can change the `number` into some string that gets executed by bash, but just in case.

Reply:
So if we use a whitelist, the python script will not make any changes to any files. The commit action will then not commit anything to the repo. So even though the workflow will trigger, we will not be building anything. 

The `site_admin` property though is a good double-check

Reply:
Ok I don't really like this new method:
Because the script will still run when any change is made on any issue -> although nothing will happen in unrelated issues, it still feels bad because we can do the check much earlier